<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardFeedDetails</class>
 <widget class="QWidget" name="StandardFeedDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="m_lblParentCategory">
     <property name="text">
      <string>Parent category</string>
     </property>
     <property name="buddy">
      <cstring>m_cmbParentCategory</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="m_cmbParentCategory">
     <property name="toolTip">
      <string>Select parent item for your feed.</string>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Type</string>
     </property>
     <property name="buddy">
      <cstring>m_cmbType</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="m_cmbType">
     <property name="toolTip">
      <string>Select type of the standard feed.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Encoding</string>
     </property>
     <property name="buddy">
      <cstring>m_cmbEncoding</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="m_cmbEncoding">
     <property name="toolTip">
      <string>Select encoding of the standard feed. If you are unsure about the encoding, then select &quot;UTF-8&quot; encoding.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="m_lblTitle">
     <property name="text">
      <string>Title</string>
     </property>
     <property name="buddy">
      <cstring>m_txtTitle</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="LineEditWithStatus" name="m_txtTitle" native="true"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="m_lblDescription">
     <property name="text">
      <string>Description</string>
     </property>
     <property name="buddy">
      <cstring>m_txtDescription</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="LineEditWithStatus" name="m_txtDescription" native="true"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>URL</string>
     </property>
     <property name="buddy">
      <cstring>m_txtUrl</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="LineEditWithStatus" name="m_txtUrl" native="true"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Fetch metadata</string>
     </property>
     <property name="buddy">
      <cstring>m_btnFetchMetadata</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="m_btnFetchMetadata">
       <property name="text">
        <string>Fetch it now</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LabelWithStatus" name="m_lblFetchMetadata" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="m_lblIcon">
     <property name="text">
      <string>Icon</string>
     </property>
     <property name="buddy">
      <cstring>m_btnIcon</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QToolButton" name="m_btnIcon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select icon for your feed.</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelWithStatus</class>
   <extends>QWidget</extends>
   <header>labelwithstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_cmbParentCategory</tabstop>
  <tabstop>m_cmbType</tabstop>
  <tabstop>m_cmbEncoding</tabstop>
  <tabstop>m_btnFetchMetadata</tabstop>
  <tabstop>m_btnIcon</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
